src/doc/book/documentation.md

```diff
@@ -460,8 +460,9 @@ not actually pass as a test.
 ```
 
 The `no_run` attribute will compile your code, but not run it. This is
-important for examples such as "Here's how to start up a network service,"
-which you would want to make sure compile, but might run in an infinite loop!
+important for examples such as "Here's how to start up a network service",
```

Comment:
Having the comma inside the quotations is correct here, as the docs are written in American English. (I remember seeing this specified somewhere...)